autoload/clang_format.vim

```diff
@@ -64,7 +64,7 @@ endfunction
 
 function! s:make_style_options() abort
     let extra_options = s:build_extra_options()
-    return printf("'{BasedOnStyle: %s, IndentWidth: %d, UseTab: %s%s}'",
+    return printf("\"{BasedOnStyle: %s, IndentWidth: %d, UseTab: %s%s}\"",
```

Comment:
@SimplyKnownAsG I found that `shellescape()` has a problem when `shellslash` is set to on. Could you try out `s:shellescape()` defined in `autoload/clang_format.vim` instead?